collect_app/src/main/java/org/odk/collect/android/preferences/screens/AdminPreferencesFragment.java

```diff
@@ -133,4 +139,10 @@ public void preventOtherWaysOfEditingForm() {
         FormEntryAccessPreferencesFragment fragment = (FormEntryAccessPreferencesFragment) getFragmentManager().findFragmentById(R.id.preferences_fragment_container);
         fragment.preventOtherWaysOfEditingForm();
     }
+
+    private void setProjectColorSummary() {
+        Spannable summary = new SpannableString("■");
+        summary.setSpan(new ForegroundColorSpan(Color.parseColor(currentProjectProvider.getCurrentProject().getColor())), 0, summary.length(), 0);
+        findPreference("project_color").setSummary(summary);
```

Comment:
This is very cool. Again, I think it could work with a `SummaryProvider` though.

Reply:
Done.